RadarSDK/RadarTrackingOptions.m

```diff
@@ -75,8 +75,8 @@ + (RadarTrackingOptions *)responsive {
     options.showBlueBar = NO;
     options.useStoppedGeofence = YES;
     options.stoppedGeofenceRadius = 200;
-    options.useMovingGeofence = YES;
-    options.movingGeofenceRadius = 200;
+    options.useMovingGeofence = NO;
```

Comment:
no more moving geofence for responsive?

Reply:
Not needed anymore. Low power location manager will keep the process awake while moving.

Reply:
Is it worth ripping out `useMovingGeofence` since none of the 3 presets use it now? Or worth keeping around for a custom option.
🤔

Reply:
Let's keep it for consistency with Android and in case we ever need or want to add it back.